### ISSUE TITLE
unit tests for fix16_exp

### ISSUE THREAD

**Issue by LeotomasMC** [PetteriAimonen/libfixmath#43]
It appears the functions in the `fix16_exp` file dont have unit tests, despite the wiki stating that there are unit tests. i did notice the wiki page is quite old, and probably has some outdated information, but if there were unit tests for these methods in the past, why would they be removed?

**Comment by PetteriAimonen** [comment]
I don't know, but to me it seems there is test code covering `fix16_exp` in two places:

* https://github.com/PetteriAimonen/libfixmath/blob/master/benchmarks/benchmark.c
* https://github.com/PetteriAimonen/libfixmath/blob/master/fixtest/main.c

But only the tests added by Martin in #28 are currently being run by the GitHub workflow.

**Comment by LeotomasMC** [comment]
i saw the second file after i posted this, but neither file seem to include tests for `fix16_log`

**Comment by PetteriAimonen** [comment]
Yeah, I think that is true. I don't know more about the history, the wiki page I copied from https://code.google.com/archive/p/libfixmath/wikis/FunctionReference.wiki

I'm not actively maintaining the code, but had some projects depending on the library so wanted to keep it available.

**Comment by PetteriAimonen** [comment]
Ok, looks like the tests used to exist:
https://github.com/PetteriAimonen/libfixmath/blob/edcfdea949d0c73ee86738f3211a59baaf722101/unittests/fix16_exp_unittests.c

They were removed in pull request #33, apparently it wasn't a very good refactoring.
I didn't want to stand in anyone's way and merged it, but in the end Martin abandoned the task.
